packages/shared/lib/core/layer-2/utils/getDestinationNetworkFromAddress.ts

```diff
@@ -7,7 +7,7 @@ export function getDestinationNetworkFromAddress(networkAddress: string | undefi
         return network?.name
     }
 
-    const chainConfigurations = network?.chainConfigurations.filter(isIscpChain)
+    const chainConfigurations = network?.chains.filter(isIscpChain)
```

Comment:
we could rename to chains also 
```suggestion
    const chains = network?.chains.filter(isIscpChain)
```